server/src/test/java/io/spine/server/event/EventBusShould.java

```diff
@@ -28,12 +28,12 @@
 import io.spine.base.EventContext;
 import io.spine.base.Responses;
 import io.spine.envelope.EventEnvelope;
+import io.spine.server.BoundedContext;
 import io.spine.server.event.enrich.EventEnricher;
 import io.spine.server.storage.StorageFactory;
-import io.spine.server.storage.StorageFactorySwitch;
 import io.spine.test.EventTests;
-import io.spine.test.Tests;
-import io.spine.test.Tests.MemoizingObserver;
+import io.spine.test.Observers;
```

Comment:
Not sure when we got that created. 

For a pretty long time we have `StreamObservers.<T>memoizingObserver()`.